tests/functionality/opflow/selfcheck.cpp

```diff
@@ -11,11 +11,12 @@ struct OpflowFunctionalityTestParameters {
   std::string solver = "";
   std::string model = "";
   std::string network = "";
-  bool is_loadloss_active;
-  bool is_powerimb_active;
-  std::string gen_bus_voltage_string;
+  bool is_loadloss_active = false;
+  bool is_powerimb_active = false;
```

Comment:
Do these also have OPFLOW default values?

Reply:
Not really. There are some related variables (e.g. "include_loadloss_variables") that have defaults that could be use to infer that default value, but nothing direct.

Reply:
Don't know the LoE but it would be nice to have these determine their default values in the same way as other options

Reply:
I could add these as default values in the OPFLOWOptions struct, but that may be above my pay grade.

Reply:
No problem